### PR TITLE
Spelling corrections everywhere

### PR DESCRIPTION
**Pull request by mjz19910**

### CODE REVIEW

**Comment by creator1creeper1** [comment]
Wasn't the first commit already merged in #11666?

File: Userland/Demos/Fire/Fire.cpp [hunk 1]
```diff
@@ -134,7 +134,7 @@ void Fire::timer_event(Core::TimerEvent&)
     if (phase > 1)
         phase = 0;
 
-    /* Paint our palettized buffer to screen */
+    /* Paint our palletized buffer to screen */
```
Comment (Dexesttp) [inline Userland/Demos/Fire/Fire.cpp:137]: That'sa mis-correction? <https://en.wiktionary.org/wiki/palettize>

File: Tests/AK/TestTypeTraits.cpp [hunk 2]
```diff
@@ -39,8 +39,8 @@
 #define EXPECT_VARIADIC_TRAIT_FALSE(trait, ...) \
     static_assert(!trait<__VA_ARGS__>)
 
-enum class Enummer : u8 {
-    Dummmy,
+enum class Enumer : u8 {
```
Comment (creator1creeper1) [inline Tests/AK/TestTypeTraits.cpp:42]: Is this supposed to be read as "Enumer[ation]" (in which case your fix would be correct) or as "Enum" -> "Enummer" as in "Run" -> "Runner" (in which case your fix would be incorrect)?

**Review by creator1creeper1 — CHANGES_REQUESTED** [review]

File: Userland/Demos/Fire/Fire.cpp (hunk 1 above)
Comment (creator1creeper1) [reply Userland/Demos/Fire/Fire.cpp:137]: Funnily enough, this was already merged. So this PR should probably revert that change.

**Comment by creator1creeper1** [comment]
According to `CONTRIBUTING.md`, "the official project language is American English", so are all the `s` -> `z` changes actually correct?

File: Userland/Libraries/LibJS/Tests/builtins/Temporal/TimeZone/TimeZone.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ describe("normal behavior", () => {
         expect(Object.getPrototypeOf(timeZone)).toBe(Temporal.TimeZone.prototype);
     });
 
-    test("canonicalizes time zone name", () => {
```
Comment (linusg) [inline Userland/Libraries/LibJS/Tests/builtins/Temporal/TimeZone/TimeZone.js:28]: This is correct.

**Review by linusg — COMMENTED** [review]
Please squash these commits, it makes no sense to have "Everywhere", and then some some specific ones.

**Comment by linusg** [comment]
> According to CONTRIBUTING.md, "the official project language is American English", so are all the s -> z changes actually correct?

Yes - AE prefers z, BE prefers s.

**Review by linusg — APPROVED** [review]
Thanks again! That's a lot of typos 😅

**Review by creator1creeper1 — APPROVED** [review]

**Pull request merged**